main.go

```diff
@@ -128,6 +129,7 @@ Usage:
   ios batterycheck [options]
   ios tunnel start [options] [--pair-record-path=<pairrecordpath>]
   ios tunnel ls [options]
+  ios deviceinfo [options] (display | lockdown)
```

Comment:
@diegoperini does it make sense to merge this with the ios info command?

Reply:
It does but it's also not a straightforward merge since the output is entirely different and we don't know who we are gonna make unhappy if we don't keep the old one. What's more is deviceinfo is a real service, named after the one listening on the device and it has/will have more domains (like display, lockdown) in the future. The merge will be a never ending problem as we introduce new domains. I'd be in favor of freezing the old one in time and deprecating it.

@dmissmann Any thoughts?

Reply:
Removed "deviceinfo" command and merged the behavior into "info". 

Usage:
```shell
# Prints same output as before, fetching the data from lockdown
ios info

# Same as above but more verbose invocation
ios info lockdown

# Newly added, fetches the data from RSD services over the tunnel
# Address and rsd port are mandatory, prints a useful error message when omitted
ios info display --address=<tunnelAddress> --rsd-port=<rsdPort>
```